contrib/static-opus/Makefile

```diff
@@ -0,0 +1,182 @@
+##
```

Comment:
So nice to read a clean makefile :)

Reply:
Thanks! Indeed; they're a pleasure to write if they can be kept simple.